dotCMS/src/main/java/com/dotmarketing/business/PermissionBitAPIImpl.java

```diff
@@ -395,18 +395,7 @@ public boolean doesUserHavePermission(final Permissionable permissionable, int p
 		}
 		String inode ="";
         List<String> ids= new ArrayList<String>();
-        String s = permissionable.toString();
-        
-        if(s.contains("formId=")){
-                String[] st = s.split(",");
-                for(String str : st){
-                        str = str.trim();
-                        if(str.contains("formId=")){
-                                inode = str.substring(7);
-                        }
-                }
-        }
-        
+
         if(inode != ""){

```

Comment:
Might something be broken with this change? This condition `if(inode != "")` will never be reached. Two things:
1) String `inode` is compared with != instead of equals
2) `inode` will always be ""